### DISCUSSION TITLE
Is it possible to create version constraints for access to packages?

### DISCUSSION THREAD

**Discussion by ericges** [vtsykun/packeton#251] [Q&A]
Hello community :)

I want to restrict certain users of accessing e.g. the next major version of a given package.
Can this be set up?

**Comment by vtsykun** [comment]
Yes, you can restrict versions using composer constraint in the Access matrix form 
 
![Selection_050](https://github.com/vtsykun/packeton/assets/21358010/e7058a06-bec2-4a60-a46b-2bd2536ef0d9)
![Selection_048](https://github.com/vtsykun/packeton/assets/21358010/256868e0-7861-43c6-b060-81b83e27a7f5)
